jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/pom.xml

```diff
@@ -547,7 +547,7 @@
                   <artifactInstaller>
```

Comment:
@sutaakar Should the Cargo container.id be changed to wildfly14x as well? If so, then we need to change also this package:
https://github.com/kiegroup/jbpm/tree/master/jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/src/test/resources/arquillian/wildfly11x

Reply:
@MarianMacik the wildfly14x id is not yet available

Reply:
@sutaakar @MarianMacik what about this? https://codehaus-cargo.github.io/cargo/WildFly+14.x.html

If not, what can we do about this?

Reply:
@adrielparedes Yes, it is available now in latest version. I already created a PR to raise it in Kie parent - https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/854
Anyway it should work even with old wildfly11x container id.